examples/offline_inference/structured_outputs.py

```diff
@@ -57,17 +57,12 @@ class CarDescription(BaseModel):
 
 # Guided decoding by Grammar
 simplified_sql_grammar = """
-    ?start: select_statement
-
-    ?select_statement: "SELECT " column_list " FROM " table_name
-
-    ?column_list: column_name ("," column_name)*
-
-    ?table_name: identifier
-
-    ?column_name: identifier
-
-    ?identifier: /[a-zA-Z_][a-zA-Z0-9_]*/
+root ::= select_statement
+select_statement ::= "SELECT" column "from" table "where" condition
+column ::= "col_1" | "col_2"
+table ::= "table_1" | "table_2"
+condition ::= column "=" number
+number ::= "1" | "2"
```

Comment:
This actually results in something like `SELECTcol_1fromtable_1wherecol_2=2`

I pulled it from our tests. We handle it OK for testing purposes there, but it shouldn't be what we provide as an example.